src/liapunovspectrum.jl

```diff
@@ -116,7 +137,7 @@ function liap_jetcoeffs!{T<:Number}(eqsdiff!, t0::T, x::Vector{Taylor1{T}},
 
         # Equations of motion
         eqsdiff!(t0, xaux, dx)
-        stabilitymatrix!( eqsdiff!, t0, xaux[1:dof], δx, δdx, jac )
+        stabilitymatrix!( eqsdiff!, t0, xaux[1:dof], δx, dδx, jac )
         @inbounds dx[dof+1:nx] = jac * reshape( xaux[dof+1:nx], (dof,dof) )
```

Comment:
I'm thinking that we could further improve allocations if we avoid the use of `reshape`, which allocates a new array, but haven't figured out yet a nice way to do this